test.js

```diff
@@ -1,8 +1,22 @@
-var mdls = require('./index.js')
+'use strict';
+const mdls = require('./index.js');
 
-mdls('./index.js', function(err, data) {
-  console.log(err, data)
-})
+(async () => {
```

Comment:
i'm not super familiar with async/await yet. Why did you wrap the test in this call?

Reply:
`await` can unfortunately only be used within an async function, so one can make an async IIFE around it like that to get it working.

While this may seem to negate the benefits, particularly as one has a lot of `await` commands, there is no need for further nesting of callbacks. But there is a need for at least one async function.